platform/darwin/src/MGLFeature.h

```diff
@@ -186,12 +187,27 @@ MGL_EXPORT
  #### Related examples
  See the <a href="https://www.mapbox.com/ios-sdk/maps/examples/runtime-multiple-annotations/">
  Dynamically style interactive points</a> example to learn how to initialize
- `MGLPointFeature` objects and add it them your map.
+ `MGLPointFeature` objects and add them to your map.
  */
 MGL_EXPORT
 @interface MGLPointFeature : MGLPointAnnotation <MGLFeature>
 @end
 
+/**
+ An `MGLPointFeatureCluster` object associates a point shape (with an optional
+ identifier and attributes) and represents a point cluster.
+ 
+ @see `MGLCluster`
+ 
+ #### Related examples
+ See the <a href="https://www.mapbox.com/ios-sdk/maps/examples/clustering/">
+ Clustering point data</a> example to learn how to initialize
+ clusters and add them to your map.
+ */
```

Comment:
<img width="374" alt="screen shot 2019-01-14 at 12 41 27 pm" src="https://user-images.githubusercontent.com/1198851/51129945-cae7df80-17f9-11e9-97c3-6f51d8af2253.png">

Jazzy currently puts this class (and `MGLEmptyFeature`) into the “Other Classes” category — to customize this, edit [`jazzy.yml`](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/ios/jazzy.yml).

Reply:
Perfect - thanks for this.